src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -242,6 +532,131 @@ module snabb-softwire-v2 {
       }
     }
 
+
+    list instance {
+      description
+       "Provides configuration for specific instances of the lwAFTR.
+        These configuration options will only affect the specific lwaftr
+        with the given name specified in the name leaf. The other options
+        not present in this list are shared amongst all instances.";
+
+      key "device";
+
+      leaf device {
+        type string;
+        description "PCI device name.";
```

Comment:
Probably needs more description here; i.e. that if no "external-interface" device is specified, that this device is for on-a-stick operation; and that if "external-interface" is specified that this interface will be for internal v6 traffic only; right?

It's a bit of a shame to have to document this in such detail; would be nice if we could figure out a way to make the configuration a bit more self-explanatory.  Maybe we should key on some abstract instance identifier instead and then just have all relevant instance-specific configuration be inside the thing :/